sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -48,13 +48,13 @@
 
 // Downsize factor for video stream
 #ifndef VIEWVIDEO_DOWNSIZE_FACTOR
-#define VIEWVIDEO_DOWNSIZE_FACTOR 4
+#define VIEWVIDEO_DOWNSIZE_FACTOR 1
```

Comment:
Any specific reason to change the defaults here?
How will that affect current configurations for e.g. ARDrone, Bebop?
Also if changing defaults, please update the module xml description accordingly.